__tests__/Unit/Components/Tasks/FilterModal.test.tsx

```diff
@@ -10,9 +10,9 @@ describe('FilterModal', () => {
     test('renders the modal with correct title and buttons', () => {
         render(
             <FilterModal
-                tabs={[Tab.ASSIGNED, Tab.IN_PROGRESS]}
+                tabs={[Tab.ASSIGNED, Tab.IN_PROGRESS, Tab.OVERDUE]}
                 onSelect={mockOnSelect}
-                activeTab={Tab.ASSIGNED}
+                activeTab={Tab.OVERDUE}
```

Comment:
why active tab is overdue instead of assigned

Reply:
Here i am testing that if overdue tab is clicked , then it should become active.